gradle/libs.versions.toml

```diff
@@ -0,0 +1,50 @@
+[versions]
```

Comment:
Yet another file format.

Reply:
tom's obvious minimal language

Reply:
(we've already checked these in elsewhere and there's nothing to really learn here but I was curious too - their arguments are at the end of this design doc https://github.com/gradle/gradle/files/5646826/2020-12-05-Central.declaration.of.dependencies.Shared.externally.pdf )